src/xcode/ENA/ENA/Source/Services/Risk/Provider/Model/__tests__/RiskProvidingConfigurationTests.swift

```diff
@@ -107,31 +116,55 @@ final class RiskProvidingConfigurationTests: XCTestCase {
 		// Test the case when last detection was performed recently
 		// There should be no need to do the detection again.
 		let lastDetection = Calendar.current.date(byAdding: DateComponents(hour: -1), to: Date()) ?? .distantPast
-		XCTAssertFalse(config.shouldPerformExposureDetection(lastExposureDetectionDate: lastDetection))
+		XCTAssertFalse(config.shouldPerformExposureDetection(activeTracingHours: 42, lastExposureDetectionDate: lastDetection))
 	}
 
 	func testShouldPerformExposureDetection_LastDetectionNow() {
 		// Test the case when last detection was performed at this instant
 		// There should be no need to do the detection again
 		let now = Date()
-		XCTAssertFalse(config.shouldPerformExposureDetection(lastExposureDetectionDate: now, currentDate: now))
+		XCTAssertFalse(config.shouldPerformExposureDetection(activeTracingHours: 42, lastExposureDetectionDate: now, currentDate: now))
 	}
 
 	func testShouldPerformExposureDetection_LastDetectionFuture() {
 		// Test the case when last detection was performed in the future.
 		// This is not valid, and a detection should be performed.
-		XCTAssertTrue(config.shouldPerformExposureDetection(lastExposureDetectionDate: Date().addingTimeInterval(10000)))
+		XCTAssertTrue(config.shouldPerformExposureDetection(activeTracingHours: 42, lastExposureDetectionDate: Date().addingTimeInterval(10000)))
 	}
 
 	func testShouldPerformExposureDetection_LastDetectionDistantPast() {
 		// Test the case when last detection was performed in the past
 		// Detection is necessary
-		XCTAssertTrue(config.shouldPerformExposureDetection(lastExposureDetectionDate: .distantPast))
+		XCTAssertTrue(config.shouldPerformExposureDetection(activeTracingHours: 42, lastExposureDetectionDate: .distantPast))
 	}
 
 	func testShouldPerformExposureDetection_NilLastDetection() {
 		// Test the case when last detection not performed at all
 		// Detection is necessary
-		XCTAssertTrue(config.shouldPerformExposureDetection(lastExposureDetectionDate: nil))
+		XCTAssertTrue(config.shouldPerformExposureDetection(activeTracingHours: 42, lastExposureDetectionDate: nil))
+	}
+
+	func testShouldPerformExposureDetection_afterInstall() {
+		// Test the case when the tracing hasn't been active long enough
+		// Detection is not necessary
+		XCTAssertFalse(config.shouldPerformExposureDetection(activeTracingHours: 7, lastExposureDetectionDate: nil))
```

Comment:
should we add a test where the active tracing hours are below 24 but there is a lastExposureDetectionDate set? I am not sure if that can happen

Reply:
This case could never happen